app/decorators/campaign_decorator.rb

```diff
@@ -0,0 +1,17 @@
+class CampaignDecorator < ApplicationDecorator
+  include ActionView::Helpers::AssetTagHelper
+  include ActionView::Helpers::UrlHelper
+
+  def sidebar_image
+    return unless active?
+
+    img = image_tag(object.sidebar_image, class: "block w-100 h-auto radius-default", width: 1000, height: 420)
```

Comment:
**question/nitpick:** how do we feel about including this markup in the decorator? Will it be confusing for future debugging to find where these classes and styles are being applied since they're not in the view?

Reply:
Excellent point @atsmith813, I didn't think about this during my spike. The decorator now takes an options hash as argument and passes it through to `image_tag`:

```ruby
<%= campaign.sidebar_image(class: "block w-100 h-auto radius-default", width: 1000, height: 420) %>
```